pallene/parser.lua

```diff
@@ -79,6 +80,33 @@ function Parser:loop_end()
     self.loop_depth = self.loop_depth - 1
 end
 
+function Parser:is_inside_region()
+    local result = false
+    if #self.regions > 0 then
+        local region = self.regions[#self.regions]
+        result = not region[2]
+    end
+    return result
+end
+
+-- The ranges are inclusive.
```

Comment:
I think it would be nice if the comment made it clear that the purpose of these functions is to implement the Pallene to Lua translator. Similartly to how the comment before loop_begin and loop_end says that the purpose is to catch invalid break statements.